src/main.rs

```diff
@@ -119,40 +137,13 @@ async fn handle_streamer_message(
         contracts,
     )
     .await?;
-    Ok(streamer_message.block.header.height)
-}
 
-fn init_tracing() -> DefaultSubcriberGuard {
-    let mut env_filter = EnvFilter::new("near_lake_framework=info,indexer_events=info");
-
-    if let Ok(rust_log) = env::var("RUST_LOG") {
-        if !rust_log.is_empty() {
-            for directive in rust_log.split(',').filter_map(|s| match s.parse() {
-                Ok(directive) => Some(directive),
-                Err(err) => {
-                    tracing::warn!(
-                        target: crate::LOGGING_PREFIX,
-                        "Ignoring directive `{}`: {}",
-                        s,
-                        err
-                    );
-                    None
-                }
-            }) {
-                env_filter = env_filter.add_directive(directive);
-            }
-        }
-    }
+    let time_since_the_epoch = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .expect("Something went wrong.");
 
-    let (non_blocking_writer, _guard) = tracing_appender::non_blocking(std::io::stderr());
-
-    let subscriber = tracing_subscriber::FmtSubscriber::builder()
-        .with_env_filter(env_filter)
-        .with_writer(non_blocking_writer)
-        .finish();
-
-    DefaultSubcriberGuard {
-        subscriber_guard: tracing::subscriber::set_default(subscriber),
-        writer_guard: _guard,
-    }
+    LATEST_BLOCK_HEIGHT.set(streamer_message.block.header.height.try_into().unwrap());
+    LATEST_BLOCK_TIMESTAMP.set(time_since_the_epoch.as_secs_f64());
```

Comment:
allows us track when the last block was indexed. We use it to figure out if the indexer is running or stalling

Reply:
We take the `LATEST_BLOCK_HEIGHT` from the streamer so we know what height the indexer has last seen. In case the indexer halts, we can go to `/probe` and see where it is at. I renamed `LATEST_BLOCK_HEIGHT` -> `LAST_SEEN_BLOCK_HEIGHT`

Also, i've removed `LATEST_BLOCK_TIMESTAMP` and instead just have`LATEST_BLOCK_TIMESTAMP_DIFF` instead which just simply records the difference between the current processing block timestamp and time of the server.